modules/move-etcd-different-disk.adoc

```diff
@@ -10,20 +10,60 @@ You can move etcd from a shared disk to a separate disk to prevent or resolve pe
 
 The Machine Config Operator (MCO) is responsible for mounting a secondary disk for {product-title} {product-version} container storage.
 
+[NOTE]
+====
+This procedure only supports device names for the following device types:
+
+* SCSI or SATA- `/dev/sd*`
++
+* Virtual - `/dev/vd*`
++
+* NVMe - `/dev/nvme*[0-9]\*n*`
+====
+
+A script for `/etc/find-secondary-device`, similar to the following sample, is encoded in the machine configuration file. Decode and replace the device name in the script according to your environment.
```

Comment:
If I understand this correctly, the script is not encoded into the machine configuration file until that step is done by the reader. I also don't think we need to decode anything unless I'm misunderstanding how machine configuration files work (which is absolutely possible, haha).

I still think that creating the script and base64-encoding it should be the second step of this procedure directly following the `lsblk` verification step.

As I understand it, essentially, the reader needs to take the contents of the script and encode them in base64. That encoded string is then included directly in the `etcd-mc.yml` file. This is the  `source: data:text/plain;charset=utf-8;base64,<lots_of_base64_noise>` string in the YAML. Personally, I think we should leave the actual base64-encoded script out of the docs, as this ensures that the reader _must_ follow the documentation instead of just copying and pasting something that may or may not work in unexpected ways.

**TL;dr:** I'd write something like the following as the second step of the procedure:

```adoc
. Encode the following script in base64, noting the output:
+
[source,bash]
----
#!/bin/bash
set -uo pipefail

for device in <device_type_glob>; do # <1>
[...]
----
<1> Replace `<device_type_glob>` with a shell glob for your block device type. For SCSI or SATA drives, use `/dev/sd*`; for virtual drives, use `/dev/vd*`; for NVMe drives, use `/dev/nvme*[0-9]\*n*`.
```

Reply:
Yes, sounds good to me.